pam/pam_test/pam-module-transaction-dummy_test.go

```diff
@@ -0,0 +1,218 @@
+package pam_test
+
+import (
+	"testing"
+
+	"github.com/msteinert/pam"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ModuleTransactionDummy_SetItem(t *testing.T) {
```

Comment:
Ok, I saw the `_` in some go projects, but and the linter didn't warn me but I didn't like much it either. It just made things more readable, but as per your other suggestion this can definitely be changed!